internal/storage/postgres/watering_plan/watering_plan.go

```diff
@@ -104,6 +104,13 @@ func WithTrailer(trailer *entities.Vehicle) entities.EntityFunc[entities.Waterin
 	}
 }
 
+func WithCanellationNote(cancellationNote string) entities.EntityFunc[entities.WateringPlan] {
```

Comment:
Typo: `Cancellation`

```suggestion
func WithCancellationNote(cancellationNote string) entities.EntityFunc[entities.WateringPlan] {
```